Apps/VisionOS/VisionOS/MainScreen.swift

```diff
@@ -68,7 +68,12 @@ struct MainScreen: View {
                     CustomerIO.shared.flush()
                 }
             case .profileAttributes:
-                Text("Profile Attribute")
+                ProfileAttributesView { attribute in
+                    CustomerIO.shared.profileAttributes[attribute.key] = attribute.value
```

Comment:
Thinking out loud, for demo how about setting `profileAttributes` same way as we show in our [docs](https://customer.io/docs/sdk/ios/tracking/identify/#identify-again) ? I mean, though I am clear with what the snippet is doing but it also gives an impression that I can set `profileAttribute` one property at a time. 

I would like to hear others thoughts on this!

Reply:
Thank you for the call out! Will fix this one. I agree with you